.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+<PROJECT_ROOT>/node_modules/resolve/test/resolver/malformed_package_json/package.json
```

Comment:
Looks like the only workaround currently https://github.com/browserify/resolve/issues/271#issuecomment-1021307415